developer/src/kmc-kmn/test/test-wasm-uset.ts

```diff
@@ -0,0 +1,70 @@
+import 'mocha';
+import sinon from 'sinon';
+import chai, { assert } from 'chai';
+import sinonChai from 'sinon-chai';
+import { Compiler } from '../src/main.js';
+import { KMCMP_ERROR_HAS_STRINGS, KMCMP_ERROR_SYNTAX_ERR, KMCMP_ERROR_UNSUPPORTED_PROPERTY, KMCMP_FATAL_OUT_OF_RANGE, UnicodeSetError } from '../src/compiler/compiler.js';
+chai.use(sinonChai);
+
+describe('Compiler class', function() {
+  let consoleLog: any;
+
+  beforeEach(function() {
+    consoleLog = sinon.spy(console, 'log');
+  });
+
+  afterEach(function() {
+    consoleLog.restore();
+  });
```

Comment:
I don't think we need sinon.spy in this unit. As we should never be `console.log`ging directly from any of our funcs anyway.

Reply:
OK.